.github/hypertest.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+function finish() {
+  make logs stop
+}
+
+trap finish EXIT
+
+make start wait-healthy
+
+docker run --rm --network host --mount "type=bind,source=$(pwd)/test/hypertest/,destination=/tests" hydrofoil/hypertest:_0.3.2 --baseUri http://localhost:8080/
```

Comment:
Not currently running anything, also seeing:

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$2 (file:/hydra-validator/node_modules/@hydrofoil/hypertest/jdeploy-bundle/app.hypermedia.testing.dsl.cli-1.0.0-SNAPSHOT-shaded.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$2
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

/cc @tpluscode

Reply:
Yes, I'm aware of that. Haven't investigated yet but it's something coming from the internals of the DSL code.

Reply:
Ok, so the warning is unrelated to the scenario not being found?

Reply:
Yes. I think the version 0.3.2 did not like to have the tests mounted.

Please try `latest` where I switched to recommending `volumes` as suggested by my colleague. I will tag that as 0.4 soon

```
version: "3"

services:
  e2e-tests:
    image: "hydrofoil/hypertest"
    network_mode: "host"
    environment:
      BASE_URI: "http://localhost:8080/"
    volumes:
      - ./test/hypertest:/tests
```